doc/architecture/decisions/0021-BLAST-sequence-formats.md

```diff
@@ -0,0 +1,94 @@
+# 21. BLAST sequence formats
+
+Date: 2022-10-17
+
+## Status
+
+Draft
+
+## Context
+This document summarises different sequence formats supported by Ensembl BLAST
+
+## Supported formats
+We support the most common sequence formats for any BLAST program - Fasta format or sequence as plain text
+
+## Decision
+Like other blast search providers we also support 2 ways of input - by pasting sequence in the input box or by uploading a file.
+
+- Input sequence to be accepted in either Fasta format characters or as plain text format (bare sequences). (Sequence IDs may be supported in the future)
+- Sequence should contain minimum 5 characters so that blast can process it properly.
+- No more than 30 sequences can be submitted at a time to prevent overloading JDispatcher.
```

Comment:
This is misleading. The reader may think that we aren't allowing submission of more than 30 jobs at once (to avoid overloading JDispatcher), when in fact, it's x number of sequences (I think we raised it to 50? at least that's what is showing in the form on the new site) times y number of selected species (we haven't defined it; but I think the maximum was either 20 or 30), which makes for up to a maximum of 1000-1500 jobs in a submission.

Reply:
Can you recall what the maximum number of species allowed is Jyo ?

Reply:
MAX_BLAST_SPECIES_COUNT = 25;
MAX_BLAST_SEQUENCE_COUNT = 50;